covigator/dashboard/figures/lineages.py

```diff
@@ -17,15 +17,16 @@
 
 class LineageFigures(Figures):
 
-    def get_lineages_plot(self, data_source: str, countries=None, lineages=None):
+    def get_lineages_plot(self, data_source: str, countries=None, lineages=None, time_period=14):
         logger.debug("Getting data on samples by country...")
         data = self.queries.get_accumulated_lineages_by_country(
             data_source=data_source, countries=countries, lineages=lineages)
         graph = dcc.Markdown("""**No data for the current selection**""")
         if data is not None and data.shape[0] > 0:
             logger.debug("Prepare plot on samples by lineage...")
             lineages = list(data.sort_values("cumsum", ascending=False).lineage.unique())
-
+            data.set_index('date', drop=False)
```

Comment:
I think this line is not doing anything, as `set_index()` does not affect the DataFrame on which it is called unless `inplace=True` is passed.

Reply:
Good hint. I thought I had already removed the line